src/main/java/az/edu/turing/domain/dao/impl/memory/PassengerDaoInMemory.java

```diff
@@ -0,0 +1,48 @@
+package az.edu.turing.domain.dao.impl.memory;
+
+import az.edu.turing.domain.dao.inter.PassengerDao;
+import az.edu.turing.domain.entity.PassengerEntity;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+public class PassengerDaoInMemory extends PassengerDao {
+
+    public static final List<PassengerEntity> PASSENGERS = new ArrayList<>();
+
+    @Override
+    public PassengerEntity save(PassengerEntity passengerEntity) {
+        PASSENGERS.add(passengerEntity);
```

Comment:
write if condition,passenger  may be exists

Reply:
that candition is not written in this class, I write that in serviceimpl class